electron-builder.json

```diff
@@ -1,5 +1,9 @@
 {
   "publish": [{
+      "provider": "generic",
+      "url": "http://localhost:8000"
+    },
```

Comment:
this should be removed once we are done with testing the feature, but added as a convenience

Reply:
Is there a Jira ticket or something to make sure we don't forget and accidentally include this in a release? This is a local privilege escalation vulnerability.

Reply:
definitely! good call!

Added https://mattermost.atlassian.net/browse/MM-38300

@devinbinnie I put it in triage so it keeps showing up until we are done with testing